CHANGELOG.md

```diff
@@ -6,7 +6,7 @@
 # 📣 What does this dbt package do?
 - Produces modeled tables that leverage Reddit Ads data from [Fivetran's connector](https://fivetran.com/docs/applications/reddit-ads) in the format described by [this ERD](https://fivetran.com/docs/applications/reddit-ads#schemainformation) and builds off the output of our [Reddit Ads source package](https://github.com/fivetran/dbt_reddit_ads_source).
 - Enables you to better understand the performance of your ads across varying grains:
-  - Providing an account, campaign, ad group, keyword, ad, utm and search level reports.
+  - Providing an account, campaign, ad group, keyword, ad, utm and search level reports. 
```

Comment:
There is no keyword report. We should probably remove this from the package details.

Reply:
thanks for catching, updated